modules/services/sql/rds.tf

```diff
@@ -31,6 +31,30 @@ resource aws_db_instance mysql_rds {
   vpc_security_group_ids = [aws_security_group.rds.id]
 }
 
+
+## This is exactly the same as the master
+resource aws_db_instance mysql_rds_replicas {
+  count = var.number_of_replicas
+  identifier = "${var.customer_name}-${var.environment}-openedx-replica-${count.index}"
+  instance_class = var.instance_class
+  engine_version = var.engine_version
+
+  storage_type = "gp2"
+  backup_retention_period = 14
+
+  publicly_accessible = false
+  storage_encrypted = true
+  kms_key_id = aws_kms_key.rds_encryption.arn
+  auto_minor_version_upgrade = false
+  multi_az = true
```

Comment:
Same comment here -- `multi_az` should be configurable for the replicas too. And Cloudera doesn't need their read replica to be `multi_az`, just a single AZ is fine.